app/presenters/layouts/student_top_nav_presenter.rb

```diff
@@ -24,16 +26,22 @@ def logo_url
 
     private
 
+    def notifications?
+      return false if current_user.blank?
+
+      current_user.notifications.where(read_at: nil).any?
```

Comment:
@bodhish Again, use the model's `unread` scope?

Reply:
Fixed in 15b24ff7c18d657c53f1d1f5eb9d08e834ee5d9d